src/pages/home/report/ReportActionCompose.js

```diff
@@ -204,7 +209,12 @@ class ReportActionCompose extends React.Component {
             // There could be other animations running while we trigger manual focus.
             // This prevents focus from making those animations janky.
             InteractionManager.runAfterInteractions(() => {
-                this.textInput.focus();
+                // Keyboard is not opened after Emoji Picker is closed
+                // SetTimeout is used as a workaround
+                // https://github.com/react-native-modal/react-native-modal/issues/114
+                setTimeout(() => {
+                    this.textInput.focus();
+                }, 100);
```

Comment:
We're really trying to avoid using setTimeouts whenever possible, could we maybe try the 

```
this.props.navigation.addListener('transitionEnd', () => {
    if (this.textInput) {
        this.textInput.focus();
    }
});
```